static/docs/get-started/connect-code-and-data.md

```diff
@@ -2,28 +2,32 @@
 
```

Comment:
Re-ordered this entire page and re-worded some paragraphs;

And made sure the contents of https://dvc.org/doc/tutorial/define-ml-pipeline include all this info (and more) as well